drivers/modem/gsm_ppp.c

```diff
@@ -35,6 +35,8 @@ LOG_MODULE_REGISTER(modem_gsm, CONFIG_MODEM_LOG_LEVEL);
 #define GSM_RECV_MAX_BUF                30
 #define GSM_RECV_BUF_SIZE               128
 #define GSM_ATTACH_RETRY_DELAY_MSEC     1000
+#define GSM_REGISTER_DELAY_MSEC         1000
+#define GSM_REGISTER_TIMEOUT            60
```

Comment:
Please make it configurable via Kconfig with default value because I had cases when registration time takes about +/- 180 sec so I would like to have the possibility to override this timeout easily.